app/views/middleware_topology/show.html.haml

```diff
@@ -10,14 +10,15 @@
         %svg.kube-topology
           %g.MiddlewareServer{:transform => "translate(21, 21)"}
             %circle{:r => "17"}
-            %image{:height => "20", :width => "20", :x => "-9", "xlink:href" => image_path('/assets/svg/vendor-wildfly.svg'), :y => "-9"}
+            %image{:height => "20", :width => "20", :x => "-10", :y => "-10", "xlink:href" => image_path('/assets/svg/vendor-wildfly.svg')}
         %label
           = _("Middleware Servers")
       %kubernetes-topology-icon{tooltipOptions, :kind => "MiddlewareDeployment"}
         %svg.kube-topology
           %g.MiddlewareDeployment{:transform => "translate(21, 21)"}
             %circle{:r => "17"}
-            %image{:height => "18", :width => "18", :x => "-9", "xlink:href" => image_path('/assets/100/middleware_deployment.png'), :y => "-9"}
+            -# product-report
+            %text{:x => "0", :y => "8", :class => "product-report glyph"} &#xE603;
```

Comment:
@abonas The 3 SVGs for Middleware were made by @pilhuhn before I created the font icons. We won't need them any longer.